groundcontrol.kv

```diff
@@ -557,49 +557,48 @@
 
 <TouchNumberInput>:
     textInput:textInput
+    cols: 1
+    size: root.size
+    pos: root.pos
+    TextInput:
+        size_hint_y:.25
+        font_size: self.height - 5
+        id:textInput
     GridLayout:
-        cols: 1
-        size: root.size
-        pos: root.pos
-        TextInput:
-            size_hint_y:.25
-            font_size: self.height - 5
-            id:textInput
-        GridLayout:
-            cols: 3
-            Button:
-                text: "1"
-                on_release: root.addText("1")
-            Button:
-                text: "2"
-                on_release: root.addText("2")
-            Button:
-                text: "3"
-                on_release: root.addText("3")
-            Button:
-                text: "4"
-                on_release: root.addText("4")
-            Button:
-                text: "5"
-                on_release: root.addText("5")
-            Button:
-                text: "6"
-                on_release: root.addText("6")
-            Button:
-                text: "7"
-                on_release: root.addText("7")
-            Button:
-                text: "8"
-                on_release: root.addText("8")
-            Button:
-                text: "9"
-                on_release: root.addText("9")
-            Button:
-                text: "."
-                on_release: root.addText(".")
-            Button:
-                text: "0"
-                on_release: root.addText("0")
-            Button:
-                text: "Done"
-                on_release: root.done()
\ No newline at end of file
+        cols: 3
```

Comment:
This change is unrelated to the issue, but fixes a warning that was showing up when the popup was created